packages/pages/src/scaffold/modules/templates.ts

```diff
@@ -69,10 +70,9 @@ export const indexCssCode = (useTailwind: boolean): string => {
   return useTailwind
     ? `.tailwind {
   @tailwind base;
+  @tailwind components;
```

Comment:
Your other change shouldn't be here if you merged

Reply:
ah just merged